spec/graphql/base_type_spec.rb

```diff
@@ -39,7 +39,6 @@
     it "resets connection types" do
       # Make sure the defaults have been calculated
       obj_edge = obj_type.edge_type
-      obj_conn = obj_type.connection_type
```

Comment:
The issue here is a typo on line 46: it should assert `refute_equal(obj_conn, obj_2.connection_type)`